src/evaluators/response/PHAL_ResponseSquaredL2DifferenceSide_Def.hpp

```diff
@@ -40,17 +55,32 @@ ResponseSquaredL2DifferenceSideBase(Teuchos::ParameterList& p, const Teuchos::RC
 
   layout->dimensions(dims);
 
+  std::string sideSetNameForMetric =
+      (plist->isParameter("Is Side Set Planar") && plist->get<bool>("Is Side Set Planar")) ?
+          sideSetName + "_planar" :
+          sideSetName;
+
   if (fieldDim>0)
   {
-    metric = decltype(metric)(Albany::metric_name + "_" + sideSetName, dl_side->qp_tensor);
+    metric = decltype(metric)(Albany::metric_name + "_" + sideSetNameForMetric, dl_side->qp_tensor);
     this->addDependentField(metric);
   }
 
   sourceField = decltype(sourceField)(fname,layout);
-  w_measure   = decltype(w_measure)(Albany::weighted_measure_name + "_" + sideSetName, dl_side->qp_scalar);
+  w_measure   = decltype(w_measure)(Albany::weighted_measure_name + "_" + sideSetNameForMetric, dl_side->qp_scalar);
   scaling     = plist->get("Scaling",1.0);
 
   this->addDependentField(sourceField);
+
+  rmsScaling = plist->isParameter("Root Mean Square Error Field Name");
+  if(rmsScaling) {
+    rootMeanSquareField = decltype(rootMeanSquareField)(plist->get<std::string>("Root Mean Square Error Field Name"),layout);
+    this->addDependentField(rootMeanSquareField);
+    rmsScaling = true;
```

Comment:
This line is redundant. You already set `rmsScaling=true` if you're in the if block.

Reply:
good catch, I'll remove the line.